Blog/views.py

```diff
@@ -146,6 +147,12 @@ class PostCreate(LoginRequiredMixin, generic.CreateView):
     def form_valid(self, form):
         form.instance.user = self.request.user
         form.instance.created = datetime.datetime.now()
+        slug = slugify(form.instance.title)
+        same_slugs_count = Post.objects.filter(slug__icontains=form.instance.slug).count()
+        if 0 < same_slugs_count:
+            # whitespace to let slugify create a hyphen
+            slug += slugify(" " + str(same_slugs_count))
```

Comment:
Mhhh, is it just me or does the whitespace not get transformed to a hyphen?

```
    150         slug = slugify(form.instance.title)
    151         same_slugs_count = Post.objects.filter(slug__icontains=slug).count()
    152         if 0 < same_slugs_count:
    153             # whitespace to let slugify create a hyphen
    154             import ipdb; ipdb.set_trace();
--> 155             slug += slugify(" " + str(same_slugs_count))
    156         form.instance.slug = slug
    157         return super().form_valid(form)
    158 
    159 
    160 def settings(request):

ipdb> same_slugs_count
1
ipdb> slugify(" " + str(same_slugs_count))
'1'
```

Reply:
What about just appending `-x` to the slug?

```
ipdb> slug
'foobar'
ipdb> same_slugs_count
1
ipdb> slug += '-' + str(same_slugs_count)
ipdb> slug
'foobar-1'
```

Reply:
Mh, I don't know why GitHub thinks this comment is outdated. Did you address this issue?

Reply:
I changed the code accordingly